src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -1,18 +1,25 @@
 package uk.gov.nationalarchives.tdr.api.graphql.fields
 
-import java.util.UUID
+import java.sql.Timestamp
+import java.util.{Date, UUID}
```

Comment:
`java.util.Date` is pretty much deprecated now because it was difficult to work with. You [should always use classes in the `java.time` package](https://codeblog.jonskeet.uk/2017/04/23/all-about-java-util-date/) (which was introduced in Java 8) unless you're dealing with legacy code. Unless there's some reason we have to use it here? 😢

Reply:
Redundant import